ios-trivia-game/ViewControllers/InviteViewController.swift

```diff
@@ -41,7 +48,49 @@ class InviteViewController: UIViewController {
         self.inviteTableView.dataSource = self
         self.inviteTableView.estimatedRowHeight = 100
         self.inviteTableView.rowHeight = UITableViewAutomaticDimension
+        
+        refreshInvites()
+        
+        self.inviteTableView.addSubview(self.refreshControl)
+    }
 
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func onTapInviteTableViewCell(_ sender: UITapGestureRecognizer) {
```

Comment:
Oh OK.. Seems to make more sense to use didSelectRow.. Unless you're planning to do some custom stuff here with the gesture recognizer.. Like triple-tap in order to do some custom stuff, which you probably can't achieve with didSelect.